contrib/kubernetes/manifests/registry-s3-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-keyfile
+type: Opaque
+data:
```

Comment:
I think it would be clearer to say this:
```yaml
data:
  accesskey: {base64-encoded access key}
  secretkey: {base64-encoded secret key}
  region: {base64-encoded region name}
  bucket: {base64-encoded bucket name}
```